docker/logstash/logstash.conf

```diff
@@ -0,0 +1,13 @@
+input {
+  file {
+    path => "/opt/libra/data/libra.log"
+  }
+}
+
+output {
+  amazon_es {
+      hosts => ["https://search-validator-logging-tzrcylcyokg64ze7vatqjwmuba.us-west-2.es.amazonaws.com"]
```

Comment:
I'd rather not have config like this baked in the docker image. Can we pass the whole config file in at runtime rather?